api/output_policy.go

```diff
@@ -68,15 +66,11 @@ func (d *OutputPolicyError) buildSamplePolicy() (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("failed to unescape request URL characters: %v", err)
 	}
-	apiAddrPrefix := fmt.Sprintf("%sv1/", d.VaultClient.config.Address)
+	apiAddrPrefix := fmt.Sprintf("%sv1/", d.vaultServerAddress)
```

Comment:
Is `vaultServerAddress` used anywhere else? If not, maybe it's possible to get rid of it and just take the string after `"v1"` as the path.